manifests/params.pp

```diff
@@ -4,19 +4,26 @@
   case $::osfamily {
     'Debian': {
       case $::operatingsystemmajrelease {
-        12.04: {
+        '12.04': {
           $net_ipv4_tcp_rmem = '4096 87380 16777216'
           $net_ipv4_tcp_wmem = '4096 65536 16777216'
+          $java_package = 'openjdk-7-jre-headless'
```

Comment:
can you also provide defaults for 16.04?

Reply:
ah stupid me. I assume the default block works for 16.04.

Reply:
The module previously claimed to support 16.04, and it looks like that was being done via the default block.

I don't mind duplicating the default block for 16.04 though if that's clearer?